src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -100,6 +103,9 @@ public PaxTarEntry(TarEntryType entryType, string entryName, IEnumerable<KeyValu
         /// <summary>
         /// Initializes a new <see cref="PaxTarEntry"/> instance by converting the specified <paramref name="other"/> entry into the PAX format.
         /// </summary>
+        /// <exception cref="ArgumentException"><para>Cannot convert a PAX Global Extended Attributes entry.</para>
+        /// <para>-or-</para>
+        /// <para>The entry type of <paramref name="other"/> is not supported for conversion to the PAX format.</para></exception>
```

Comment:
Doesn't this sentence encompasses the one above? Is there a reason to have them separate?

Reply:
The first sentence refers to the type of `other` itself, PaxGlobalExtendedAttributesTarEntry, which cannot be converted at all, while the second sentence refers to the entry type of `other`, where the format of the entry can be V7, Ustar, Pax (excluding PaxGEA) or Gnu.